lib/Backends/NNPI/NNPIDeviceManager.cpp

```diff
@@ -63,8 +64,8 @@ NNPIDeviceManager::NNPIDeviceManager(
     std::shared_ptr<NNPIDeviceOptions> deviceOptions, NNPIAdapter adapter,
     unsigned numInferenceWorkers)
     : DeviceManager(config), numWorkersPerFunction_(numInferenceWorkers),
-      deviceId_(config_.deviceID), adapter_(adapter),
-      device_(NNPI_INVALID_NNPIHANDLE), deviceOptions_(deviceOptions) {
+      deviceId_(config_.deviceID), device_(NNPI_INVALID_NNPIHANDLE),
+      deviceOptions_(deviceOptions), adapter_(adapter) {
```

Comment:
@omromano @jsubag 
This ordering was also an intentional change I made to your last PR. Without it there is a build warning/error due to incorrect initialization order. Same comment as above - please pull what's in github master into your tree so we don't try to revert it on the next PR. Thanks!

Reply:
@arunm-git 
Sorry we missed that. Will make sure it updated in our code (and not reverted in the next PR).